hr_timesheet_sheet_week_start_day/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+=================================
+Hr Timesheet Sheet Week Start Day
```

Comment:
My title was a suggestion as current name is a little bit awkward, that's all.
Module names is fine to me.
(non blocking)